ui/vic-ui-h5c/vic/src/vic-app/src/app/summary-portlet/container-portlet/container-portlet.component.ts

```diff
@@ -42,21 +45,30 @@ import { VirtualMachine } from '../../vm.interface';
 export class ContainerPortletComponent implements OnInit {
     @Input() activeVm: VirtualMachine;
 
-    private containerNameLabel: string;
-    private containerImageLabel: string;
-    private containerPortmappingLabel: string;
+    public containerNameLabel: string;
+    public containerImageLabel: string;
+    public containerPortmappingLabel: string;
+
+    constructor(
+        private i18n: I18nService,
+        private gs: GlobalsService
+    ) {}
 
-    constructor() {}
+    isPluginMode() {
+        return this.gs.isPluginMode() && !(<any>this.activeVm)['is_testing'];
+    }
 
     ngOnInit() {
-        this.containerNameLabel = window.hasOwnProperty('com_vmware_vic') ?
-            com_vmware_vic.getString('container.name.label') :
+        this.containerNameLabel = this.isPluginMode() ?
+            this.i18n.translate('container.name.label') :
             'Container';
```

Comment:
The hard coded values are to be consumed in unit tests where we don't actually test i18n. In the plugin mode (in the VC environment) then the values are fed directly from the properties files.